### PR TITLE
Ignore small ESP block devices if there are multiple choices

### PR DESCRIPTION
**Pull request by hughsie**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2214191

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Review by superm1 — COMMENTED** [review]
What if they're both (or all) small? Seems unlikely but I think this would make them all get skipped. So don't you want to make sure that there is at least one bigger before you skip the first small?

**Comment by hughsie** [comment]
What about if we sorted them by size, and just chose the biggest one?

File: libfwupdplugin/fu-volume.c [hunk 1]
```diff
@@ -197,6 +197,31 @@ fu_volume_get_id(FuVolume *self)
...
 
+/**
+ * fu_volume_get_size:
+ * @self: a @FuVolume
+ *
+ * Gets the size of the block device pointed to by the volume.
+ *
+ * Returns: size in bytes, or 0 on error
+ *
+ * Since: 1.6.3
```
Comment (superm1) [inline libfwupdplugin/fu-volume.c:208]: 1.9.3?

**Review by superm1 — APPROVED** [review]

File: libfwupdplugin/fu-volume.c (hunk 1 above)
Comment (hughsie) [reply libfwupdplugin/fu-volume.c:208]: Indeed, fixed, thanks.

**Pull request merged**